cmd/dvotecli/commands/common.go

```diff
@@ -0,0 +1,26 @@
+package commands
+
+import (
+	"fmt"
+
+	"go.vocdoni.io/dvote/crypto/ethereum"
+)
+
+const MaxListIterations = 256
```

Comment:
Shouldn't you take MaxListSize from the scrutinizer package?

Reply:
No, this is the number of iterations on the total, and it's an arbitrary number. For a CLI, I think it should show the total number of elements, without requiring any pagination. 

`router.MaxListSize` is already being used in the pagination loop, so the total would be 16384 lines, already higher than must terminal buffers, but it may be needed to dump all values in a file, for example.

Does it makes sense?

Reply:
Yes, thanks!